.github/configs/evm-impl.yaml

```diff
@@ -1,3 +1,6 @@
+eels:
+  evm-bin: ethereum-spec-evm-resolver
+  x-dist: auto
```

Comment:
Attempt to reduce sporadic fails?
```suggestion
  x-dist: 8
```

Reply:
Since Peter's resolver hardening, this PR now has 8 green Github actions, I think we can skip this :slightly_smiling_face:

Reply:
Jinxed it, sporadic fail encountered during framework tests. The Github actions runner is using 4 runners with `-n auto` though, so we def shouldn't apply this change.

Reply:
Perhaps the framework tests (in `tox -e framework`) could potentially briefly stress the resolver more than filling tests (in `tox -e tests`).

Some of the framework tests `fill` by calling the transition tool's `evaluate()` directly. These won't have the normal overhead incurred when filling and might result in more concurrent requests to the resolver.

Reply:
Marked framework tests that call `fill` or `generate()` to only run in serial in `tox -e framework`, 4aaf99fdc.

Reply:
I'm not sure it's these tests that are the problem any more. Resolving for now and taking the discussion below, cf https://github.com/ethereum/execution-spec-tests/pull/792#pullrequestreview-2333121342